lib/sniffybara.rb

```diff
@@ -1,11 +1,11 @@
-require "capybara"
+require 'capybara'
```

Comment:
You might need to require `webdrivers` here too. Without it, I'm getting some failures:

```
     Selenium::WebDriver::Error::WebDriverError:
       Unable to find chromedriver. Please download the server from
       https://chromedriver.storage.googleapis.com/index.html and place it somewhere on your PATH.
       More info at https://github.com/SeleniumHQ/selenium/wiki/ChromeDriver.
```